source/change-streams/change-streams.rst

```diff
@@ -536,45 +536,23 @@ so resuming is impossible.
 Test Plan
 =========
```

Comment:
Should we move these into the README?  I think that's what our other specs do for prose tests.

Reply:
done